### ISSUE TITLE
Load and Play DeckLink input

### ISSUE THREAD

**Issue by cat2608** [SuperFlyTV/casparcg-connection#54]
Hi, using CasparCG Client I'm able to *load* and *play* a DeckLink input and the servers output is:

```
Received message from 127.0.0.1: LOADBG 1-10 DECKLINK DEVICE 1 FORMAT 1080i5000\r\n
[frame_muxer] simple 1920x1080i50.00
DeckLink SDI [1|1080i5000] Initialized
transition[empty=>DeckLink SDI [1|1080i5000]] Initialized
Sent message to 127.0.0.1:202 LOADBG OK\r\n

Received message from 127.0.0.1: PLAY 1-10\r\n
Sent message to 127.0.0.1:202 PLAY OK\r\n
```

I'm trying to use `loadbg()` and `play()` but the input is being quoted and CasparCG fails:

```
Received message from 127.0.0.1: LOADBG 1-10 "DECKLINK DEVICE 1 FORMAT 1080i5000"\r\n
No match found for supplied commands. Check syntax. Turn on log level debug for stacktrace.
Sent message to 127.0.0.1:404 LOADBG FAILED\r\n
```

What is the correct command to handle DeckLinks?
Thanks

**Comment by toontoet** [comment]
Might be related to https://github.com/SuperFlyTV/casparcg-connection/issues/52 
Also a problem caused by the extra quotes that are forced around the CLIP part.

**Comment by cat2608** [comment]
The command is being updated here: https://github.com/SuperFlyTV/casparcg-connection/blob/master/src/lib/ParamValidators.ts#L133

Our solution is using the `AMCP.CustomCommand`:

```
import { CasparCG, AMCP } from 'casparcg-connection';
const connection = new CasparCG();
connection.do(new AMCP.CustomCommand('LOADBG 1-10 DECKLINK DEVICE 1 FORMAT 108015000\r\n'));
```

**Comment by jesperstarkar** [comment]
Thank you for this issue. It should be solved together with #52. Perhaps the "clip" parameter should be renamed or extended to reflect that other content could be played/loaded. Or, it would be better to keep the Clip param, but adding extra params like "decklink", "html" etc. and have them resolve into "clip" for backwards compability?

I'm in any case going to add highlevel commands to the CasparCG object, giving us CasparCG.PlayDecklink(), CasparCG.PlayHTML() etc.
